aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -61,6 +62,18 @@ pub enum Commands {
         #[clap(short, long)]
         print_gas_schedule: bool,
     },
+    /// Print out package metadata.
```

Comment:
Can you add some usage example like from your test plan?

Reply:
done